modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenEncoding.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
For new files, you don't need line 3 as per https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license/7301#7301

Reply:
Ok. I will fix it and push a new commit.